nettrace/httpclient.go

```diff
@@ -185,18 +205,18 @@ type conntrackEntry struct {
 // Source/destination is from the client side.
 type connection struct {
 	addrTuple
-	id             TraceID
-	sockCreatedAt  Timestamp
-	connectedAt    Timestamp // for TCP this is just after handshake
-	closedAt       Timestamp
-	reused         bool
-	closed         bool
-	dialID         TraceID
-	fromResolver   bool
-	conntrack      conntrackEntry
-	totalRecvBytes uint64
-	totalSentBytes uint64
-	socketOps      []SocketOp
+	ID             TraceID
```

Comment:
Are these attributes now exported because of bbolt?
Shouldn't `dial` and `tlsTun` also have all the fields exported now?
And what about `addrTuple`, shouldn't it be exported as well?